pythonforandroid/recipes/libxml2/__init__.py

```diff
@@ -0,0 +1,31 @@
+from pythonforandroid.toolchain import (CythonRecipe, shprint,
+                                        current_directory, info, Recipe)
+from pythonforandroid.patching import will_build, check_any
+import sh
+from os.path import join
+
+
+class Libxml2Recipe(CythonRecipe):
+    version = '2.9.4'
```

Comment:
Presumably this version is much newer than the one the old toolchain used? It's hopefully fine and definitely should be targeted, but just to note it could be a source of problems.